hw03/hw03.Rmd

```diff
@@ -0,0 +1,192 @@
+    intel_pomp = sum(c(intel1_pomp,intel2_pomp,intel3_pomp,intel4_pomp,intel5_pomp,intel6_pomp,intel7r_pomp, intel8r_pomp,intel9r_pomp,intel10r_pomp), na.rm = TRUE),
+    open_pomp = sum(c(open1_pomp,open2_pomp,open3_pomp,open4_pomp,open5_pomp,open6_pomp,open7r_pomp, open8r_pomp,open9r_pomp,open10r_pomp), na.rm = TRUE)) 
+```
+
+```{r}
+green_data %>%
+  summary(green_pomp)
+```
+
+
+  4. Make plots that illustrate the distributions of the 4 POMP-scored variables.
+  
+```{r}
+ggplot(green_data) +
+  aes(x = green_pomp) +
+  geom_bar() +
+  theme_bw()
+
+ggplot(green_data) +
+  aes(x = comp_pomp) +
+  geom_bar() +
+  theme_bw()
+
+ggplot(green_data) +
+  aes(x = intel_pomp) +
+  geom_bar() +
+  theme_bw()
+
+ggplot(green_data) +
+  aes(x = open_pomp) +
+  geom_bar() +
+  theme_bw()
+```
+
+  5. Make scatterplots showing the relationships between **green reputation** and each personality trait.
+     Include trend lines for **students** and **non-students**.
+     What do these plots show?
+     
+```{r}
+green_data %>% 
+  filter(!is.na(student)) %>% 
+  ggplot() +
+  aes(y = comp_pomp,
+      x = green_pomp,
+      group = student,
+      color = student,
+      fill = student) +
+  geom_jitter()
+
+green_data %>% 
+  filter(!is.na(student)) %>% 
+  ggplot() +
+  aes(y = intel_pomp,
+      x = green_pomp,
+      group = student,
+      color = student,
+      fill = student) +
+  geom_jitter()
+  
+green_data %>% 
+  filter(!is.na(student)) %>% 
+  ggplot() +
+  aes(y = open_pomp,
+      x = green_pomp,
+      group = student,
+      color = student,
+      fill = student) +
+  geom_jitter()
+```
+     
+  6. Compare **green reputation** for students and non-students using a **rainfall plot** (bar + density + data points).
+  
+```{r}
+
+```
+  
+  
+  7. Compute a summary table of means, SDs, medians, minima, and maxima for the four total scores for students and non-students.
+  
+```{r}
+summary_green <- green_data %>% 
+  na.omit() %>% 
+  group_by(student) %>% 
+  summarize(green_mean = mean(green_pomp),
+            green_sd = sd(green_pomp),
+            green_median = median(green_pomp),
+            green_min = min(green_pomp),
+            green_max = max(green_pomp),
+            comp_mean = mean(comp_pomp),
+            comp_sd = sd(comp_pomp),
+            comp_median = median(comp_pomp),
+            comp_min = min(comp_pomp),
+            comp_max = max(comp_pomp),
+            intel_mean = mean(intel_pomp),
+            intel_sd = sd(intel_pomp),
+            intel_median = median(intel_pomp),
+            intel_min = min(intel_pomp),
+            intel_max = max(intel_pomp),
+            open_mean = mean(open_pomp),
+            open_sd = sd(open_pomp),
+            open_median = median(open_pomp),
+            open_min = min(open_pomp),
+            open_max = max(open_pomp))
+
```

Comment:
You can also shorten this by using the across function as such:

summarize(across(c("column1":"column2")